stratego_player/lib/stratego_player/impl.ex

```diff
@@ -0,0 +1,42 @@
+defmodule StrategoPlayer.Impl do
+    def connect(game_node) do
+        Node.connect(game_node) 
+        :global.sync()
+    end
+
```

Comment:
It looks as if you never actually use the result from `send_to_game`.

And in `print_board`, you send 100 requests to the server to get the value of each square.

I don't know the details of the app, but I was wondering if you could save some network traffic by having `:player_ready`, `:place_piece` and `move_piece` simply return a board representation which you could pass directly to `print_board`?